server/serverMethods.js

```diff
@@ -707,12 +711,14 @@ function ServerMethods(aLogLevel, aModules) {
   function getHangUp(aReq) {
```

Comment:
Is this always called? 
We could end up in a situation where the key isn't removed from Redis and we block a number out.

Reply:
I think we can rely on Plivo calling this when the call is dropped.